process/project_proposals.adoc

```diff
@@ -0,0 +1,118 @@
+
+=== Project stages
+
+image::project-stages.png[Project stages]
+
+=== Sandbox process
+
+Note: We have changed the sandbox proposal process to a https://docs.google.com/document/d/1IlhqnmhSzs3aTPMkp_75JPYz6LD21aK1sNPrBNruDt0/edit?usp=sharing[new lighter-weight process for Sandbox proposals].
+
+All exceptions (and "no" outcomes) are handled by the TAC. Possible "no" outcomes include "not at this time", and the project may be encouraged to re-apply after addressing issues. Timeframes are approximate, to set expectations.
+
+image::sandbox-process.png[Sandbox process]
+. *Project Proposal*
+   * Project proposed through a https://docs.google.com/forms/d/1bJhG1MuM981uQXcnBMv4Mj9yfV5_q5Kwk3qhBCLa_5A/edit[form]
+. *TAC Review*
+   * TAC reviews proposal in spreadsheet and presentation, project passes for inclusion with a simple majority vote.
+   * TAC may engage with project to ask further questions
+   * Projects may re-apply after six months
+. *Governance / legal issues*
+   * OpenSSF staff handle governance / legal issues
+   * This can happen after TAC Sandbox Sponsors step forward
+
+See the link:sandbox.md[Sandbox guidelines] for the definition of and motivation behind the OpenSSF Sandbox.
+
+=== Incubation process
+
+Note: We have https://docs.google.com/presentation/d/1J9nti4JdiwLHxY15KtkmqyfP4OgNfrLAd3vxPvFTzsc/edit?usp=sharing[streamlined the Incubation process].
+
+All exceptions (and "no" outcomes) are handled by the TAC.
+
+image::incubation-process.png[Incubation process]
+
+. *Project Proposal*
+   * Incubation proposed through a https://github.com/ossf/tac/pulls[GitHub pull request]
+   * The proposal moves to Due Diligence when a TAC member steps forward as an Incubation Sponsor.
+   * The status of outstanding Incubation proposals is reported on a monthly basis in the TAC public meeting. This highlights projects looking for sponsorship, and provides a check-in on DD progress for sponsored projects.
+   * A potential sponsor can indicate that they are interested but don't have capacity to work on DD at this time, to set a project's expectations.
+   * The TAC may agree that the project does not (yet) meet the link:graduation_criteria.adoc#incubation-stage[Incubation requirements] and give feedback on why this is the case. If the project is not already in the OpenSSF, the TAC may suggest that project apply for Sandbox instead.
+   * If a TAC Incubation Sponsor has not stepped forward within two months after the proposal PR is submitted, projects may request that their project proposal is discussed at a forthcoming TAC meeting by adding it to the https://docs.google.com/document/d/1jpoKT12jf2jTf-2EJSAl4iTdA7Aoj_uiI19qIaECNFc/edit[Working Doc]. The outcome of this is discussion is either that a sponsor steps forward, or that the TAC votes to admit the project to Sandbox, or the proposal is rejected (projects may reapply after six months). If, even after all those steps, a sponsor does not step forward, the proposal is rejected.
```

Comment:
Need at least to change the link to point to the TAC agenda: https://docs.google.com/document/d/18BJlokTeG5e5ARD1VFDl5bIP75OFPCtzf77lfadQ4f0/edit I'm not sure how stable this is though...